CHANGELOG.md

```diff
@@ -181,14 +198,72 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
+#include <seqan3/core/{algorithm => configuration/detail}/concept.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_element_debug_mode.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_element_parallel_mode.hpp>
+#include <seqan3/core/{algorithm => configuration/detail}/configuration_utility.hpp>
+#include <seqan3/core/{algorithm => configuration}/pipeable_config_element.hpp>
+
+#include <seqan3/{core/bit_manipulation => std/bit}.hpp>
+#include <seqan3/{core/bit_manipulation => utility/detail/bits_of}.hpp>
+#include <seqan3/{core/bit_manipulation => utility/detail/to_little_endian}.hpp>
+
+#include <seqan3/{core => utility}/char_operations/predicate.hpp>
+#include <seqan3/{core => utility}/char_operations/predicate_detail.hpp>
+#include <seqan3/{core => utility}/char_operations/pretty_print.hpp>
+#include <seqan3/{core => utility}/char_operations/transform.hpp>
+
+#include <seqan3/{core => utility/tuple}/common_tuple.hpp>
+
+#include <seqan3/{core/concept/core_language => utility/detail/exposition_only_concept}.hpp>
+#include <seqan3/{core/concept/tuple => utility/tuple/concept}.hpp>
+
+#include <seqan3/{core => utility}/math.hpp>
+
+#include <seqan3/{core => utility/tuple}/pod_tuple.hpp>
+
+#include <seqan3/{core => utility}/simd/concept.hpp>
+#include <seqan3/{core => utility}/simd/{ => detail/}debug_stream_simd.hpp>
+#include <seqan3/{core => utility}/simd/{simd_algorithm => algorithm}.hpp>
+#include <seqan3/{core => utility}/simd/simd.hpp>
+#include <seqan3/{core => utility}/simd/simd_traits.hpp>
+#include <seqan3/{core => utility}/simd/{view_iota_simd => views/iota_simd}.hpp>
+#include <seqan3/{core => utility}/simd/{view_to_simd => views/to_simd}.hpp>
+
+#include <seqan3/{core/tuple_utility => utility/tuple/pop_front}.hpp>
+#include <seqan3/{core/tuple_utility => utility/tuple/split}.hpp>
+
+#include <seqan3/{core => utility}/type_list/traits.hpp>
+#include <seqan3/{core => utility}/type_list/type_list.hpp>
+
+#include <seqan3/{core => utility}/type_traits/basic.hpp>
+#include <seqan3/{core => utility}/type_traits/concept.hpp>
+#include <seqan3/core/{type_traits => detail}/deferred_crtp_base.hpp>
+#include <seqan3/{core => utility}/type_traits/function{ =>_traits}.hpp>
+#include <seqan3/{core/type_traits/function => utility/detail/multi_invocable}.hpp>
+#include <seqan3/core/{type_traits/iterator => detail/iterator_traits}.hpp>
+#include <seqan3/{core => utility}/type_traits/{lazy => lazy_conditional.hpp>
+#include <seqan3/core/{type_traits/lazy => detail/is_class_template_declarable}.hpp>
+#include <seqan3/{core/type_traits/pack => utility/type_pack/traits}.hpp> 
+#include <seqan3/{core => utility}/type_traits/pre.hpp>
+#include <seqan3/core/{type_traits/range => range/type_traits}.hpp>
+#include <seqan3/core/{type_traits => detail}/template_inspection.hpp>
+#include <seqan3/{core => utility}/type_traits/{ => detail/}transformation_trait_or.hpp>
```

Comment:
```suggestion
#include <seqan3/{core => utility}/type_traits/basic.hpp>
#include <seqan3/{core => utility}/type_traits/concept.hpp>
#include <seqan3/{core => utility}/type_traits/function{ =>_traits}.hpp>
#include <seqan3/{core => utility}/type_traits/{lazy => lazy_conditional.hpp>
#include <seqan3/{core/type_traits/pack => utility/type_pack/traits}.hpp> 
#include <seqan3/{core => utility}/type_traits/pre.hpp> [deleted without replacement]
#include <seqan3/core/{type_traits/range => range/type_traits}.hpp>
```

#include <seqan3/core/{type_traits => detail}/deferred_crtp_base.hpp>
#include <seqan3/{core/type_traits/function => utility/detail/multi_invocable}.hpp>
#include <seqan3/core/{type_traits/iterator => detail/iterator_traits}.hpp>
#include <seqan3/core/{type_traits/lazy => detail/is_class_template_declarable}.hpp>
#include <seqan3/core/{type_traits => detail}/template_inspection.hpp>
#include <seqan3/{core => utility}/type_traits/{ => detail/}transformation_trait_or.hpp>

are detail header

#include <seqan3/{core => utility}/type_traits/pre.hpp> will be removed in SeqAn3.1